src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { Loader } from '../components/Loader';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { SortOrders } from '../types/SortOrder';
+
+interface FilterOptions {
+  people: Person[];
+  query: string;
+  sex: string;
+  centuries: string[];
+  sort: string;
+  order: string;
+}
+
+const getFilteredPeople = ({
+  people,
+  query,
+  sex,
+  centuries,
+  sort,
+  order,
+}: FilterOptions) => {
+  let visiblePeople = [...people];
+
+  if (query) {
+    const normalizedQuery = query.trim();
+
+    visiblePeople = visiblePeople.filter(person => (
+      person.name.toLowerCase().includes(normalizedQuery)
+      || person.fatherName?.toLowerCase().includes(normalizedQuery)
+      || person.motherName?.toLowerCase().includes(normalizedQuery)
```

Comment:
looks like we can move this logic into separated function and reuse
`.toLowerCase().includes(normalizedQuery)`